sdk/contentsafety/ai-content-safety-rest/test/public/contentSafety.spec.ts

```diff
@@ -0,0 +1,234 @@
+    }
+    assert.strictEqual(createBlockListResponse.status, "201");
+    assert.equal(createBlockListResponse.body.blocklistName, blocklistName);
+  });
+
+  it("add block items", async function () {
+    const addBlockItemsResponse = await client
+      .path("/text/blocklists/{blocklistName}:addBlockItems", blocklistName)
+      .post({
+        body: {
+          blockItems: [
+            {
+              description: "Test block item 1",
+              text: blockItemText1,
+            },
+            {
+              description: "Test block item 2",
+              text: blockItemText2,
+            },
+            {
+              description: "Test block item 3",
+              text: blockItemText3,
+            },
+          ],
+        },
+      });
+    if (isUnexpected(addBlockItemsResponse)) {
+      throw new Error(addBlockItemsResponse.body?.error.message);
+    }
+    assert.strictEqual(addBlockItemsResponse.status, "200");
+    assert.isArray(addBlockItemsResponse.body.value);
+
+    if (isRecordMode()) {
+      await sleep(30000);
+    }
+  });
+
+  it("analyze text with blocklist", async function () {
+    const analyzeTextResponse = await client.path("/text:analyze").post({
+      body: {
+        text: "I h*te you and I want to k*ll you.",
+        blocklistNames: [blocklistName],
+        breakByBlocklists: false,
+      },
+    });
+    if (isUnexpected(analyzeTextResponse)) {
+      throw new Error(analyzeTextResponse.body?.error.message);
+    }
+    assert.strictEqual(analyzeTextResponse.status, "200");
+  });
+
+  it("list text blocklists", async function () {
+    const listTextBlocklistsResponse = await client.path("/text/blocklists").get();
+    if (isUnexpected(listTextBlocklistsResponse)) {
+      throw new Error(listTextBlocklistsResponse.body?.error.message);
+    }
+    assert.strictEqual(listTextBlocklistsResponse.status, "200");
+    assert.isArray(listTextBlocklistsResponse.body.value);
+  });
+
+  it("get text blocklist", async function () {
+    const getTextBlocklistResponse = await client
+      .path("/text/blocklists/{blocklistName}", blocklistName)
+      .get();
+    if (isUnexpected(getTextBlocklistResponse)) {
+      throw new Error(getTextBlocklistResponse.body?.error.message);
+    }
+    assert.strictEqual(getTextBlocklistResponse.status, "200");
+    assert.equal(getTextBlocklistResponse.body.blocklistName, blocklistName);
+  });
+
+  it("list block items", async function () {
+    const listBlockItemsResponse = await client
+      .path("/text/blocklists/{blocklistName}/blockItems", blocklistName)
+      .get();
+    if (isUnexpected(listBlockItemsResponse)) {
+      throw new Error(listBlockItemsResponse.body?.error.message);
+    }
+    assert.strictEqual(listBlockItemsResponse.status, "200");
+    assert.isArray(listBlockItemsResponse.body.value);
+    blockItemId = listBlockItemsResponse.body.value[1].blockItemId;
+  });
+
+  it("list block items with pagination", async function () {
```

Comment:
Can you add another test to leverage the pagination helper ? Thanks https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/purview/purview-scanning-rest/samples-dev/listDatasources.ts#L28

Reply:
Add new test case "list block items with pagination helper"